src/bp/ui-studio/src/web/components/Shared/Interface/index.tsx

```diff
@@ -197,9 +198,14 @@ const SectionAction = (action: SectionAction, idx: number) => {
 }
 
 export const Toolbar = props => {
+  const children: JSX.Element[] = props.children && !props.children.length ? [props.children] : props.children
   return (
     <div className={style.toolbar}>
-      <ButtonGroup minimal={true}>{props.children}</ButtonGroup>
+      {children.map(c => (
+        <ButtonGroup key={c.key} className={c.props.className} vertical={false} minimal={true}>
+          {c}
+        </ButtonGroup>
+      ))}
     </div>
```

Comment:
The toolbar should remain simple, thus being only a ButtonGroup. I'm not a fan of having Fragments in the toolbar. If you have text to display, i'm sure it can be displayed outside of the toolbar (or implemented some other way).

Reply:
Why can't we allow multiple button groups? The problem I have here is I wan't some buttons to be on the left side of the toolbar and some others to be on the right. This gives a lot more flexibility.